clrdefinitions.cmake

```diff
@@ -15,6 +15,7 @@ if (CLR_CMAKE_TARGET_ARCH_AMD64)
   if (CLR_CMAKE_PLATFORM_UNIX)
     add_definitions(-DDBG_TARGET_AMD64_UNIX)
   endif()
+  add_definitions(-DBIT64=1)
   add_definitions(-D_TARGET_AMD64_=1)
```

Comment:
Oh, could I add BIT32 there? I tried the following structure, but it doesn't work:
```
#if defined(BIT32)
  ...
#elif defined(BIT64)
  ...
#else
  ASSERT(...)
#endif
```
I thought that both BIT32 and BIT64 are missing, but only BIT32 is missing as you pointed out.